kolibri/core/device/test/test_api.py

```diff
@@ -862,3 +864,29 @@ def test_usersyncstatus_list__unknown_non_negative_device_status(self):
         self.assertEqual(len(response.data), 1)
         self.assertEqual(response.data[0]["user"], self.user1.id)
         self.assertEqual(response.data[0]["status"], user_sync_statuses.RECENTLY_SYNCED)
+
+    def test_downloads_queryset(self):
+        last_download_removed_timestamp = datetime.datetime.now()
+        self.client.login(
+            username=self.user1.username,
+            password=DUMMY_PASSWORD,
+            facility=self.facility,
+        )
+        self._create_transfer_session(
+            active=False,
+        )
+        content_request = ContentDownloadRequest.build_for_user(self.user1)
+        content_request.contentnode_id = uuid.uuid4().hex
+        device_status = self._create_device_status("oopsie", StatusSentiment.Neutral)
+        self.syncsession1.client_instance_id = device_status.instance_id
+        self.syncsession1.save()
+        response = self.client.get(reverse("kolibri:core:usersyncstatus-list"))
+        content_request.save()
+        self.assertEqual(len(response.data), 1)
+        self.assertEqual(response.data[0]["user"], self.user1.id)
+        self.assertEqual(response.data[0]["status"], user_sync_statuses.RECENTLY_SYNCED)
+        self.assertFalse(response.data[0]["has_downloads"], True)
+        self.assertFalse(
+            response.data[0]["last_download_removed"],
+            last_download_removed_timestamp.strftime("%Y-%m-%d %H:%M:%S"),
+        )
```

Comment:
It looks like the assert suggested by Richard has not been added here

Reply:
There was indeed an issue which a test with this assertion would have illuminated.